src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -68,9 +67,6 @@ unsigned int DirectoryService::ComputeDSBlockParameters(
     }
     powDSWinners[submitter.second] = m_allPoWConns[submitter.second];
     dsWinnerPoWs[submitter.second] = m_allDSPoWs[submitter.second];
-    sortedPoWSolns.erase(
```

Comment:
@ShengguangXiao to re-order the SortPoWSoln calls, I need to remove the dependency of `sortedPoWSolns` in `DirectoryService::ComputeDSBlockParameters`.